Build-V3-Plugin-Monitoring-ios/Classes/TracesManagers/FirebasePerformanceTracesManager.swift

```diff
@@ -12,6 +12,9 @@ import WebKit
 
 public protocol FirebasePerformanceDelegate {
```

Comment:
Use Protocol extension to provide default implementations 

```Swift
public extension FirebasePerformanceDelegate {
  func performance_didFinishTraceName(for index: Int) -> String {
     return "webView_\(index)_didFinish"
   }

   func performance_DOMContentLoadedTraceName(for index: Int) -> String {
     return "webView_\(index)_DOMContentLoaded"
   }
   
  // etc

}
```

- This will simplify the implementation in client Apps (because you provide a default implementation)
- This will simplify your code, because the default implementation is separated in a clean way